### ISSUE TITLE
Deeply nested objects are hidden

### ISSUE THREAD

**Issue by ellis** [sachinraja/trpc-playground#30]
Our endpoints have some deeply nested objects, and the tRPC Playground hides them after a certain depth.  Can these be made visible?  As is, I can't use the playground for testing such endpoints.  Here's a simplified example, where the contents of `aim` are hidden:

```
{
	rows: [
		{
			keyToData: {
				SPAGT: {
					aim: [Object]
				}
			}
		}
	]
}
```

**Comment by MatsDK** [comment]
Hi, thanks for reporting this. 
This is happening because the [object-inspect](https://www.npmjs.com/package/object-inspect) package is being used in [this](https://github.com/sachinraja/trpc-playground/blob/main/packages/utils/src/index.ts#L3) file to print out the response object and it defaults the depth to 5. If we want to print the whole object this option could be set to `false` or `null`

**Comment by ellis** [comment]
Thanks @MatsDK

I thought I would try to patch that myself for a quick workaround.  I grepped `index:2` and found it in `node_modules/@trpc-playground/html/dist/assets/index.342a56e3.js`.  Appending `,depth:null` didn't have the desired effect, though, and checking the Network tab in Chrome showed that the file is actually loaded from a CDN instead of locally (`http://cdn.jsdelivr.net/npm/@trpc-playground/html@0.3.3/dist/assets/index.342a56e3.js`).

Any suggestion for a quick workaround?

Another thought: If you happen to make `depth` configurable, could you make `index` configurable too?

**Comment by MatsDK** [comment]
I don't know if the depth should be configurable, maybe it's better to set the depth to `null` by default and print the whole object.
@sachinraja what do you think?

**Comment by sachinraja** [comment]
Yeah I don't want to offer configuration for this. Let's set the depth to null or use something better for displaying the result of queries/mutations (like [`react-json-tree`](https://github.com/reduxjs/redux-devtools/tree/main/packages/react-json-tree)).

**Comment by sachinraja** [comment]
fixed in 5916cd6129040cd76cdf9d8d448bb93863e4108b
